internal/command/ssh/issue.go

```diff
@@ -97,24 +101,21 @@ func runSSHIssue(ctx context.Context) (err error) {
 		emails   string
 		rootname string
 	)
-	switch args := flag.Args(ctx); len(args) {
-	case 0:
-		// neither
-	case 1:
-		// org only
-	case 2:
+
+	orgFlag := flag.GetOrg(ctx)
+	path := flag.GetString(ctx, "path")
+
+	if orgFlag != "" && path != "" {
+		// org+email+path
+		emails = orgFlag
```

Comment:
it doesn't make sense.

seems the original command accepted 3 args (org, email and path), although usage docs says it only does 2 `issue [org] [path]`.

We need another `--email` flag, a StringSlice or whatever matches the `mail.ParseAddress()` output.

Reply:
I will try looking into the history of the command first to get some more context.

Reply:
Found the commit where it worked with 3 args, applying fix.